### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by thesleepingsage** [th-ch/youtube-music#2172]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.12

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

Windows

### Operating System Version

Windows 11

### What CPU architecture are you using?

x64

### Last Known Working YouTube Music (Application) version

3.3.6?

### Reproduction steps

1. Start app
2. Play music

### Expected Behavior

Music to not sound tinny

### Actual Behavior

Music sounds tinny

### Enabled plugins

1. Tuna OBS
2. ad blocker
3. Sponsor block

### Additional Information

This only occurs within the app.

**Comment by ArjixWasTaken** [comment]
Have you checked the windows volume mixer to see if you accidentally set the volume just for this program to a low value?

**Comment by thesleepingsage** [comment]
> Have you checked the windows volume mixer to see if you accidentally set the volume just for this program to a low value?
> 
> https://www.asus.com/support/faq/1044702/

I'd like to clarify that the issue I'm experiencing is not related to volume. If it were, I would have described the levels as quiet. Instead, the sound quality is tinny. I'm confused as to why the post title was changed to describe a problem I didn't mention having. This misrepresentation doesn't accurately reflect the issue I'm seeking help for.

**Comment by ArjixWasTaken** [comment]
Sorry about that, I indeed thought it was a typo.

But what can I do, you didn't explain much, the title was literally "[bug]: "

**Comment by thesleepingsage** [comment]
I thought I filled out every field, so that mistake was on me. I didn't realize the title was blank.

**Comment by JellyBrick** [comment]
There is no code in this application related to sound quality; I think another application is controlling this app's sound quality.

**Comment by thesleepingsage** [comment]
Yeah I am not sure. When I listen through the app everything sounds tinny and hollow. It only sounds this way through the app and not the browser. I realize the app isn't exactly an app, and also like you said it isn't touching the sound quality, but for some reason the audio quality through yt-m just sounds bad and I don't know why.

I thought it was because I was using the Windows Mixer and routing the output device to a separate cable, but even setting it back to default sounds the exact same.

Strange :(

**Comment by farsdewibs0n** [comment]
I'm 90% sure an EQ software is tinkering with the sound, try checking what program is running in the background.

If you're audio driver is Realtek, open the Realtek Audio Console and make sure the Equalizer and Environment set to None and disable Loudness Equalization.

**Comment by JellyBrick** [comment]
Are you using Discord and other voice chat applications? Windows has a feature that automatically reduces the volume when you use a calling device. Could you check your Windows setting?

**Comment by Squall-Leonhart** [comment]
> I'm 90% sure an EQ software is tinkering with the sound, try checking what program is running in the background.

there is no EQ software tinkering with the sound, something is wrong with the sample rate conversion in this app

> Are you using Discord and other voice chat applications? Windows has a feature that automatically reduces the volume when you use a calling device. Could you check your Windows setting?

totally irrelevant, it is not a volume issue.

**Comment by apo-tek** [comment]
I've experienced the same issue on the latest release and with a similar system and audio setup. In my case the culprit is the audio compressor plugin (however I did not find why it is degrading the audio this much) so it might be a good idea to give disabling it a try.
Hope this helps even if this is a pretty old issue.

**Comment by thesleepingsage** [comment]
> I've experienced the same issue on the latest release and with a similar system and audio setup. In my case the culprit is the audio compressor plugin (however I did not find why it is degrading the audio this much) so it might be a good idea to give disabling it a try. Hope this helps even if this is a pretty old issue.

At the time, I wasn't using the compressor plugin. I haven't used the software in a while, but that's the relevant info I can provide.
